cirq/ops/common_gates.py

```diff
@@ -18,6 +18,7 @@
 )
 
 import numpy as np
+import cirq
```

Comment:
Do not import `cirq` from within `cirq`. Use `from cirq import protocols` to get at `is_parameterized`.